src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -210,14 +210,15 @@ function StripePaymentForm({
       <h5 aria-level="2">
         <FormattedMessage
           id="payment.card.details.billing.information.heading"
-          defaultMessage="Billing Information"
```

Comment:
[question] does this have any effect on translations at all?

Reply:
Good point, is that something that can be done pre merge or is it a post merge chore?

Reply:
I think this can be done after merge. I think it's like:
1. this PR merges to master
2. we run the push transifex job 
3. confirm there is a new (non-translated) string in transifex with "(Required)" waiting to be translated
4. _we wait some amount of time until strings are translated_
5. we run the pull transifex job to update our files here

so there will be some amount of time where the non-EN text boxes will be out of date, but based on how I understand our system, that's to be expected